### PR TITLE
fix(MovableCoord): fire release event

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#288 

## Details

Although direction of movement is different from options.direction, MC should fire 'release' event.

## Preferred reviewers
@sculove, @netil, @mixed 

@netil it's related with `eg.flicking`

### CODE REVIEW

File: src/movableCoord.js [hunk 1]
```diff
@@ -183,6 +183,10 @@ eg.module("movableCoord", ["jQuery", eg, window, "Hammer"], function($, ns, glob
 						this._subOptions = options;
 						this._status.curHammer = hammer;
 						this._panstart(e);
+					} else if (e.isFinal && e.direction !== MC.DIRECTION_NONE &&
+						!(e.direction & this._subOptions.direction)) {
+						// Although direction of movement is different from options.direction, MC should fire 'release' event.
+						this._panend(e);
```
Comment (sculove) [inline src/movableCoord.js]: @jongmoon you'd better make `_triggerRelease` method like this
```js
_triggerRelease: function(e) {
this._setInterrupt(false);
this.trigger("release", {
	//...
});
```
and I find potential bug due to your PR.
we should also initalize `this._status.moveDistance`
thanks

Comment (jongmoon) [inline src/movableCoord.js]: @sculove OK, I also think that _panend do much~
I'm happy to help you find the potential bug related with `moveDistance`.

File: src/movableCoord.js [hunk 2]
```diff
@@ -411,7 +413,7 @@ eg.module("movableCoord", ["jQuery", eg, window, "Hammer"], function($, ns, glob
 			}
 
 			// Abort the animating post process when "tap" occurs
-			if (e.type === "tap") {
+			if (e.distance === 0 /*e.type === "tap"*/) {
```
Comment (sculove) [inline src/movableCoord.js]: you should remove tap recognizer in `_createHammer` method.

Comment (jongmoon) [inline src/movableCoord.js]: @sculove I applied it yesterday.

Comment (sculove) [inline src/movableCoord.js]: ok I got it!

**Comment by sculove** [comment]
LGTM

**Comment by mixed** [comment]
👍

**Pull request merged**